extern/nlohmann_json/README.md

```diff
@@ -1,2 +1,2 @@
 This directory holds a simple build recipe for the `nlohmann::json` library. Used in case
-`DETRAY_USE_SYSTEM_NLOHMANN` is set to `FALSE` for the build.
\ No newline at end of file
+`DETRAY_USE_SYSTEM_NLOHMANN` is set to `FALSE` for the build.
```

Comment:
What is the change it is picking up here?

Reply:
I'm guessing end-of-file consistency?